src/Components/Home.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import '../Styles/Home.scss';
 import PetContainer from './PetContainer';
+import Header from './Header';
 
 const Home = () => {
   return (
     <div>
       <section className='upperSection'>
+        <div>
```

Comment:
I think you can remove this `<div>` because you added the flex styling on the parent section in both the Home and Header components